dvc/remote/base.py

```diff
@@ -191,7 +191,7 @@ def _calculate_checksums(self, file_infos):
                 tasks,
                 total=len(file_infos),
                 unit="md5",
-                desc="Computing hashes (only done once)",
+                desc="Computing file checksum(s) (only done once)",
```

Comment:
Should probably revert per https://github.com/iterative/dvc.org/pull/933#pullrequestreview-350538870

Reply:
OK I reverted this and applied "hash" terminology as much as possible to the whole file, as well as a few others in 5019fe14. If this commit is acceptable, I can update all the other instances (there's less than 70, in 19 files.
Or I can extract further changes on this to a separate issue/PR. Cc @mroutis @efiop